collections/infrastructure/roles/powerman/README.md

```diff
@@ -14,6 +14,14 @@ powerman --on bar,foo[7,9-10]
 
 Refer to: https://linux.die.net/man/1/powerman
 
+### Force IPMI 2.0 driver
+
+Set the following variable to `true`. Default is `false`.
+
+``` yml
+enable_ipmi_lan_2_0: true
```

Comment:
Could you add `powerman_` prefix before variable name ? This convention was introduced to avoid variables colliding between roles when sys admin configured Ansible to share variables between all roles of a playbook.

Reply:
Done